WcaOnRails/.env.development

```diff
@@ -0,0 +1,10 @@
+DISABLE_BULLET=false
```

Comment:
For our devs, we should provide a file with a name like `sample.env` that is deliberately **not** picked up by dotenv, to make them aware that they explicitly need to copy/create their own config file.

This will also require updating the README accordingly.

Reply:
What's the rationale behind that?
Why not just give people a file automatically picked up, since in most cases no overrides are needed?

Reply:
Perhaps when overrides are needed, they could just copy `.env.development` and rename it `.env.development.local` which is more prioritized by `dotenv`. Otherwise they can simply stick to the defaults in `.env.development`.

Reply:
Wow, totally missed this open comment. Sorry, totally my fault that this went unmerged for so long!

@viroulep The rationale would be to actively make people aware that there's a part of the app they can configure (if wanted/needed). I feel that this is more helpful than having code that "just works" in terms of not understanding where the values come from.
But seeing that there seems to be a convenient local override mechanism, we can just go with @notrichardpeng's suggestion.

Reply:
should we mention this way of overriding in `README.md` ?

Reply:
I'd rather add a wiki page for this. The readme should only contain instructions that are immediately important for getting up and running (which should be possible without custom overrides)